ddtrace/vendor/contextvars/__init__.py

```diff
@@ -6,7 +6,7 @@
     from collections import Mapping
 
 
-__all__ = ("ContextVar", "Context", "Token", "copy_context")
+__all__ = ("ContextVar", "Context", "Token", "copy_context", "_get_context")
```

Comment:
```suggestion
__all__ = ("ContextVar", "Context", "Token", "copy_context")
```

no idea why this happened, but it can go away